### PR TITLE
[doc] document default values option

### PR DESCRIPTION
**Pull request by bomgar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +174,18 @@ implicit val residentFormat = Json.format[Resident]
 
 With the `Reads` and/or `Writes` in scope, I can then easily convert my class using `toJson` and `fromJson`
 
+In case you don't want to use an Option but a sensible default value you can use case class default parameters
```
Comment (cchantep) [inline README.md]:
```
an `Option`
```

Comment (cchantep) [reply README.md]: ", you can define default values for case class."

File: README.md [hunk 2]
```diff
@@ -174,6 +174,18 @@ implicit val residentFormat = Json.format[Resident]
...
+
+```scala
+case class Resident(name: String, age: Int, role: Role = Role.DefaultRole)
+```
+
+You need to enable DefaultValues for this to work
+```scala
```
Comment (cchantep) [inline README.md]: Blank line before code

File: README.md [hunk 3]
```diff
@@ -174,6 +174,18 @@ implicit val residentFormat = Json.format[Resident]
...
+implicit val residentFormat: Format[Resident] = Json.using[Json.WithDefaultValues].format[Resident]
+```
+
```
Comment (cchantep) [inline README.md]: Only one blank line

File: README.md (hunk 1 above)
Comment (bomgar) [reply README.md]: That doesn't sound right

**Comment by bomgar** [comment]
Fine i won't contribute.

**Pull request closed without merging**